cmake/package_dragndrop_dmg.cmake

```diff
@@ -137,6 +137,32 @@ endif ()
 
 ########################################################### SEARCHENGINES
 if(EXISTS ${SEARCH_ENGINES_DIRECTORY})
+  if(EXISTS ${SEARCH_ENGINES_DIRECTORY}/Fido)
+    install(DIRECTORY             ${SEARCH_ENGINES_DIRECTORY}/Fido
+            DESTINATION           OpenMS-${CPACK_PACKAGE_VERSION}/TOPP/SEARCHENGINES
+            COMPONENT             SearchEngine-Fido
```

Comment:
I guess we threat Fido as a "SearchEngine" so I think it is ok. Maybe in the future this could be renamed to something like "ThirdPartyTool"